pkg/get/tools.go

```diff
@@ -604,10 +604,12 @@ https://github.com/inlets/inletsctl/releases/download/{{.Version}}/{{$fileName}}
 			Name:        "terraform",
 			Version:     "1.0.0",
 			Description: "Infrastructure as Code for major cloud providers.",
-			URLTemplate: `{{$arch := .Arch}}
-
+			URLTemplate: `
+			{{$arch := ""}}
```

Comment:
We need to update the corresponding unit test for both tools.

Please could you take a look?

There are other Hashicorp tools, they may also have new arm64 binaries.

Reply:
Made the necessary changes for the terraform unit test, also added new tests for `packer`, since there wasn't any.

I checked all the available HashiCorp tools, @jsiebens did a great job :+1: , Waypoint already had the proper handling for all the architectures, Vagrant from the other hand only ships with `amd64`.